<launch>
    <node name="closedloopfbtraj"
        pkg ="plane_camera_magnet"
        type = "closedloopfbtraj"
        output ="screen"
        launch-prefix = "gdb -ex run --args">
        <param name="param_filename" value="/home/denise/catkin_ws/src/plane_camera_magnet/calib/closedloopfbpoint.yml"/>
         <param name="traj_filename" 
         value="/home/denise/catkin_ws/src/plane_camera_magnet/traj.csv"/>
    </node>
    <node name="roboclaw4listen_pub"
        pkg ="plane_camera_magnet"
        type = "roboclaw4listen_pub.py"
        output = "screen">
        <remap from="/closedloopfbpoint/roboclawcmddesired" to="/closedloopfbtraj/roboclawcmddesired"/>
    </node>
</launch>